crystal/lucky/config/log.cr

```diff
@@ -1,3 +1,3 @@
 log = Log::IOBackend.new
 log.formatter = Dexter::JSONLogFormatter.proc
-Log.dexter.configure(:info, log)
+Log.dexter.configure(:none, log)
```

Comment:
I think this whole file can just be

```
Log.dexter.configure(:none)
Avram::QueryLog.dexter.configure(:none)
```

Reply:
sure, its clarify :ok_hand: 

However,  `avram` is not used

Reply:
oh, right. Good point. 👍